README.md

```diff
@@ -155,6 +155,29 @@ https://github.com/panorama-ed/memo_wise. This project is intended to be a safe,
 welcoming space for collaboration, and contributors are expected to adhere to
 the [code of conduct](https://github.com/panorama-ed/memo_wise/blob/main/CODE_OF_CONDUCT.md).
 
+## Releasing
+
+To make a new release of `MemoWise` to Rubygems, first install the release
```

Comment:
Would it make sense to link RubyGems here? Or maybe the memo_wise RubyGems page once it's up?

Reply:
While we're talking about this line, tiny nitpick: `Rubygems` -> `RubyGems`

Reply:
Updated to fix capitalization and add link!